docs/FHIRServerUsersGuide.md

```diff
@@ -1167,87 +1167,6 @@ and then we could modify the user-defined schematron rule as follows:
-
-For incoming requests, if the servlet filter detects that the request body is encrypted (by examining the `Content-Encoding` and `AES-Salt` header values) it will decrypt the request body using the `AES/CBC/PKCS5Padding` algorthm (IBMJCEFIPS provider) along with the configured 256-bit encryption key and the IV found in the `AES-Salt` request header of the HTTP request, and then will provide the decrypted data to downstream filters and servlets (for example, the JAX-RS servlet) via the input stream.
-
-### 4.5.4 Response processing
-On the response path, if the `Accept-Encoding` header is set in the request and contains the string `aescbc256`, the servlet filter encrypts the response body using the same encryption algorithm, configured encryption key and a randomly-generated IV. The IV will be specified in the `AES-Salt` response header.
-
-The encrypted response data is then transmitted by the servlet container back to the API consumer, which decrypts the response body (perhaps via a JAX-RS ReaderInterceptor implementation), using the shared encryption key and the IV obtained from the `AES-Salt` header found in the HTTP response.
-
-### 4.5.5 Configuration
-The configuration of the encryption feature consists of the generation of an encryption key, plus the definition of a few properties in the `fhir-server-config.json` file.
-
-#### 4.5.5.1 Generate encryption key
-In order to enable the encryption feature, you'll need to generate an AES 256-bit encryption key and store it within a keystore file. This keystore file is then configured with the FHIR server and should also be shared with API consumers that need to use the encryption feature (API consumers will need to use the same encryption key).
-
-You can generate the encryption key and store it in a keystore file by using a keytool command of the following form:
-
-```
-keytool -genseckey -keystore <keystore-filename> -storetype jceks
-   -storepass <keystore-password> -keyalg AES -keysize 256
-   -alias fhirEncryptionKey -keypass <key-password>
-```
-
-When generating the encryption key and keystore file, the `storetype` should be set to `jceks`, the alias should be set to `fhirEncryptionKey`, the key algorithm must be `AES`, and the key size must be `256` (bits). You can choose any value you want for the `keystore-filename`, `keystore-password`, and `key-password` values.
-
-#### 4.5.5.2 Configuration properties
-
-In addition to generating the encryption key and storing it within the keystore file, you also need to define a set of configuration properties like the following within the `fhir-server-config.json` file:
-
-```
-{
-    "fhirServer":{
-        ...
-        "encryption":{
-            "enabled":true,
-            "keystoreLocation":"resources/security/fhirkeys.jceks",
```

Comment:
we should remove this file now (from both the client and the server)

Reply:
File is removed in the latest commit.